bump-periodics.sh

```diff
@@ -10,12 +10,14 @@ PERIODICS_DIR="${2}/ci-operator/config/shiftstack/shiftstack-ci"
 
 OLD_VERSION="4.$(( ${NEW_VERSION#4.} - 1 ))"
 OLD_OLD_VERSION="4.$(( ${OLD_VERSION#4.} - 1 ))"
+OUT_OF_SUPPORT_VERSION="4.$(( ${OLD_VERSION#4.} - 5 ))"
 
-OLD_PERIODIC="${PERIODICS_DIR}/shiftstack-shiftstack-ci-main__periodic-${OLD_VERSION}.yaml" 
-NEW_PERIODIC="${PERIODICS_DIR}/shiftstack-shiftstack-ci-main__periodic-${NEW_VERSION}.yaml" 
+OLD_PERIODIC="${PERIODICS_DIR}/shiftstack-shiftstack-ci-main__periodic-${OLD_VERSION}.yaml"
+NEW_PERIODIC="${PERIODICS_DIR}/shiftstack-shiftstack-ci-main__periodic-${NEW_VERSION}.yaml"
```

Comment:
Am I blind? What changed in these two lines?

Reply:
Trimmed empty chars at the end of the line.

Reply:
weird. Github usually highlights those (like it did in the following lines)